compiler/rustc_arena/src/lib.rs

```diff
@@ -108,7 +108,7 @@ impl<T> Default for TypedArena<T> {
             // alloc() will trigger a grow().
             ptr: Cell::new(ptr::null_mut()),
             end: Cell::new(ptr::null_mut()),
-            chunks: RefCell::new(vec![]),
+            chunks: Default::default(),
```

Comment:
Personally I'm not really a fan of putting `Default::default` everywhere unless it *really* shortens code, but that's just a nit.

Reply:
There's a tradeoff here: duplication vs explicitness. I took this from DropArena but I think you are right here.

Reply:
Both are fine with me, it's mostly just a personal prefferrence :).